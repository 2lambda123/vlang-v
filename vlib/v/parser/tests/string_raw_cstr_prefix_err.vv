fn main() {
    cr_str := rc'hello'
    println(cr_str) 
}
