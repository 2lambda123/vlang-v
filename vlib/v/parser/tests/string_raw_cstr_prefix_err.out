vlib/v/parser/tests/string_raw_cstr_prefix_err.vv:2:15: error: cannot use `r` (raw string) and `c` (c string) together
    1 | fn main() {
    2 |     cr_str := rc'hello'
      |               ~~
    3 |     println(cr_str) 
    4 | }
